_bibliography/peer-pubs.bib

```diff
@@ -101,4 +101,20 @@ @article{10944238
   doi      = {10.1109/MCSE.2025.3555023},
   issn     = {1558-366X},
   month    = {April}
-}
\ No newline at end of file
+}
+
+@article{yehudi_etal_2025_research_software_lifecycles,
+  title = {Research {{Software Lifecycles}} and {{Stages}}},
+  author = {Yehudi, Yo and Cashman, Mikaela and Goedicke, Michael and Hasselbring, Wilhelm and Katz, Daniel S. and M{\"u}ller, Sebastian and Goble, Carole and Jay, Caroline},
+  year = 2025,
+  month = dec,
+  journal = {Electronic Communications of the EASST},
+  volume = {85 },
```

Comment:
```suggestion
  volume = {85},
```

Reply:
As per style, the value of `number` (which is following after the value of `volume` in this style) doesn't work well for this "number" (*deRSE25 - Selected Contributions of the 5th Conference for Research Software Engineering in Germany*). 

<img width="1533" height="225" alt="grafik" src="https://github.com/user-attachments/assets/5097a465-0467-480e-a821-5eec4613f4f0" />

To introduce the extra space before the opening bracket, I introduced it in volume.

Alternatively, we could remove `number` completely. The Zotero parser parses this into number, but looking at the journal, it really only has volumes, and volume names (which this is).

Your choice, @danielskatz :).

Reply:
seems ok